docs/installation/Installation.md

```diff
@@ -63,8 +63,10 @@ You can check example usage at [Nussknacker Quickstart repository](https://githu
 
 All configuration options are described in [Configuration](../installation_configuration_guide/DesignerConfiguration.md).
 
-Some of them can be configured using environment variables, which is mostly helpful in Docker setup.
-In the table below there are all environment variables used in Nussknacker image. $NUSSKNACKER_DIR is a placeholder pointing to Nussknacker installation directory.
+Some of them can be configured using already predefined environment variables, which is mostly helpful in Docker setup.
+In the table below there are all predefined environment variables used in Nussknacker image. $NUSSKNACKER_DIR is a placeholder pointing to Nussknacker installation directory.
```

Comment:
1. "In the table below there are all" -> "The table below shows all"
2. "in Nussknacker image" -> "in the Nussknacker image"
3. "to Nussknacker installation directory" -> "to the Nussknacker installation directory"
4. $NUSSKNACKER_DIR -> `$NUSSKNACKER_DIR` (use ` around the env)